### DISCUSSION TITLE
Integration of Checkpoint IPS to Security Onion

### DISCUSSION THREAD

**Discussion by RoyTM99** [Security-Onion-Solutions/securityonion#8452] [Unsupported Versions] [answered]
Good Day Team,

The reason for this query is to validate if Security Onion can parse without problems the IPS product of Checkpoint technology, the idea is to validate with you if something similar happens as with the Cisco ASA or the same Checkpoint firewall which we have integrated by syslog, without any problem. Or, would I have to develop some ingest parser?

Security Onion Version
2.3.90

I remain attentive to your comments.
Kind Regards
Roy T.

**Comment by Sahale** [comment, marked as answer]
You can use [CheckPoint filebeat module](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-checkpoint.html) to parse data.
[Here](https://docs.securityonion.net/en/2.3/filebeat.html#modules) you can read how to enable modules.
